docs/delegates/1e183e43b68b39762801cd47c1363ce1c5a4093fcd70d33c3ad5513e66a79ae8/README.md

```diff
@@ -33,3 +33,5 @@ I would be very happy if you would review [my website](http://mtaylan.inova.comp
   - [https://forum.unikname.com/t/uns-node-monitoring-via-telegram/977](https://forum.unikname.com/t/uns-node-monitoring-via-telegram/977)
 - **[July 28 / 2021 ]** UNS_NODE Monitoring via Discord
   - [https://forum.unikname.com/t/uns-node-monitoring-via-discord/1139](https://forum.unikname.com/t/uns-node-monitoring-via-discord/1139)
+- **[September 12 / 2021 ]** Airdrop-1
+  - [AirDrop-1](https://twitter.com/muzaffer_taylan/status/1437051296327077894)
```

Comment:
You may sort them in descending order.
That may be better
Just an advise :)

Reply:
Order of contributions has been rearranged. Thank you very much for your advise :)